backend/Controllers/LoginController.cs

```diff
@@ -33,12 +33,11 @@ IUsersRepo userRepo
         return new UnauthorizedResult();
       }
 
-      var encodedUsernamePassword = authHeader.Substring("Basic ".Length).Trim();
-      string usernamePassword = AuthHelper.Base64Decode(encodedUsernamePassword);
+      AuthHelper.UsernamePassword usernamePassword = AuthHelper.GetUsernameAndPasswordfromAuthheader(authHeader);
```

Comment:
If you put `using {...}.AuthHelper` at the top, this can just be
```suggestion
      UsernamePassword usernamePassword = GetUsernameAndPasswordfromAuthheader(authHeader);
```